pkg/gedis/types/directory/tfgrid_node_2.go

```diff
@@ -86,3 +88,27 @@ func (e TfgridNodePublicIface1TypeEnum) String() string {
 	}
 	return "UNKNOWN"
 }
+
+// MarshalJSON implements json.Marshaler interface
+func (e TfgridNodePublicIface1TypeEnum) MarshalJSON() ([]byte, error) {
```

Comment:
I don't get why this is done manually, if this type in the schema file is already an Enum, the marshalers should be generated for u automatically!